Fix flaky tests

- https://github.com/uki00a/carol/runs/704032173
- https://github.com/uki00a/carol/runs/703998960

```shell
test Application#onExit ... FAILED (96ms)
    at Object.receive (file:///home/runner/work/carol/carol/transport.ts:44:13)
    at async ChromeImpl.readLoop (file:///home/runner/work/carol/carol/chrome.ts:241:13)
error: Uncaught ConnectionReset: Connection reset by peer (os error 104) 
    at unwrapResponse ($deno$/ops/dispatch_minimal.ts:63:11)
test custom executablePath ... Makefile:7: recipe for target 'test' failed
    at Object.sendAsyncMinimal ($deno$/ops/dispatch_minimal.ts:106:10)
    at async Object.write ($deno$/ops/io.ts:65:18)
    at async BufWriter.flush (https://deno.land/std@0.53.0/io/bufio.ts:479:25)
    at async writeFrame (https://deno.land/std@0.53.0/ws/mod.ts:144:3)
make: *** [test] Error 1
##[error]Process completed with exit code 2.
```

:thinking: :cry:

After rewring code base using puppeteer (#55), another error sometimes appears...
https://github.com/uki00a/carol/runs/1610745787

```shell
running 11 tests
test Application#evaluate (part 1) ... ok (6762ms)
test Application#evaluate (part 2) ... ok (5154ms)
test Application#exposeFunction (part 1) ... ok (5175ms)
test Application#exposeFunction (part 2) ... ok (5174ms)
thread 'main' panicked at 'Error 'Trying to work with closed connection' contains boxed error of unknown type', cli/errors.rs:35:7
stack backtrace:
   0: rust_begin_unwind
             at /rustc/7eac88abb2e57e752f3302f02be5f3ce3d7adfb4/library/std/src/panicking.rs:483
   1: std::panicking::begin_panic_fmt
             at /rustc/7eac88abb2e57e752f3302f02be5f3ce3d7adfb4/library/std/src/panicking.rs:437
   2: deno::errors::get_error_class_name::{{closure}}
   3: deno::errors::get_error_class_name
   4: deno_core::ops::json_serialize_op_result
   5: <futures_util::future::future::Map<Fut,F> as core::future::future::Future>::poll
   6: <futures_util::future::future::Inspect<Fut,F> as core::future::future::Future>::poll
   7: <futures_util::future::future::Map<Fut,F> as core::future::future::Future>::poll
   8: <futures_util::stream::futures_unordered::FuturesUnordered<Fut> as futures_core::stream::Stream>::poll_next
   9: deno_core::runtime::JsRuntime::poll_event_loop
  10: <core::future::from_generator::GenFuture<T> as core::future::future::Future>::poll
  11: <core::future::from_generator::GenFuture<T> as core::future::future::Future>::poll
  12: deno::test_command::{{closure}}
  13: <core::future::from_generator::GenFuture<T> as core::future::future::Future>::poll
  14: tokio::runtime::Runtime::block_on
  15: deno::main
note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.
make: *** [test] Error 101
test Application#onExit ... Makefile:8: recipe for target 'test' failed
Error: Process completed with exit code 2.
```

--- a/carol_test.ts
+++ b/carol_test.ts
@@ -54,7 +54,6 @@ import {
   join,
 } from "./deps.ts";
 import {
-  delayWhenRunningInCI,
   startFileServer,
   test,
   testApp,
@@ -177,7 +176,6 @@ test("Application#onExit", async () => {
     called = true;
   });
 
-  await delayWhenRunningInCI();
   await app.exit();
 
   assert(called);

--- a/carol_test.ts
+++ b/carol_test.ts
@@ -53,11 +53,7 @@ import {
   fromFileUrl,
   join,
 } from "./deps.ts";
-import {
-  startFileServer,
-  test,
-  testApp,
-} from "./test_util.ts";
+import { startFileServer, test, testApp } from "./test_util.ts";
 import { EvaluateError, launch } from "./mod.ts";
 import type { Application } from "./mod.ts";
 

--- a/test_util.ts
+++ b/test_util.ts
@@ -21,14 +21,6 @@ export function testApp(
   });
 }
 
-function delay(ms: number): Promise<void> {
-  return new Promise<void>((resolve, _) =>
-    setTimeout(() => {
-      resolve();
-    }, ms)
-  );
-}
-
 export function test(name: string, fn: () => Promise<void>): void {
   Deno.test({
     ignore: chromeDoesNotExist,

--- a/test_util.ts
+++ b/test_util.ts
@@ -15,18 +15,10 @@ export function testApp(
     const app = await launch(options);
     try {
       await fn(app);
-      await delayWhenRunningInCI();
     } finally {
       await app.exit();
     }
   });
-}
-
-// FIXME: Tests are flaky on CI. As a workaround, We put a short delay.
-export async function delayWhenRunningInCI(): Promise<void> {
-  if (Deno.env.get("CI")) {
-    await delay(5000);
-  }
 }
 
 function delay(ms: number): Promise<void> {

